TRANSLATING.md

```diff
@@ -0,0 +1,219 @@
+# Guide for Translators
+
+This guide is written to provide instructions from scratch for contributing to the translation of the Jamulus application to other languages.
+
+There are two parts:
+
+1. Instructions for getting set up, which only needs to be done once.
+
+2. The workflow for updating and contributing translations as a part of the preparation for a release of Jamulus.
+
+----
+## Setting up
```

Comment:
This section feels more like a summary of the details that follow, rather than "Setting up", which implies a one-off activity.  I first though it was the "Part 1" from the "There are two parts".

Reply:
Good points, I’ll restructure slightly.